src/Auxiliary/Aux_Check_Parameter.cpp

```
@@ -1159,8 +1172,38 @@ void Aux_Check_Parameter()
    if ( OPT__INIT == INIT_BY_FILE )
       Aux_Message( stderr, "WARNING : currently we don't check MIN_DENS for the initial data loaded from UM_IC !!\n" );
 
-   } // if ( MPI_Rank == 0 )
 
+#  elif ( WAVE_SCHEME == WAVE_GRAMFE ) // #  if ( WAVE_SCHEME == WAVE_FD )
+
+   const double dt_fluid_max = 0.4;
```

- [x] Is this an empirical result?
- [x] If it is unconditionally unstable, should we raise an error instead of a warning since this PR has passed the testing stage?

---

It is an empirical result, but the upper bound is actually a bit lower. If we raise an error, maybe we should raise an error for the corresponding check of the finite-difference scheme too?

---

Agree